package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "gutenberg",
-	"version": "3.1.1",
+	"version": "3.2.0",
```

Comment:
Sigh.. this wasn't meant to be included 😞 Only to help trigger the failing E2E locally.

Reply:
Sorry I missed it too 😢